generators_and_lazy_evaluation.ipynb

```diff
@@ -0,0 +1,667 @@
+{
```

Comment:
<p>Typos:</p><p>Wait<strong> those are</strong> actually not the first ten primes(two is missing), they <strong>are the</strong> next ten! That is because we used the same generator, that we already used in the previous cell. Try this by running the cell <strong>again</strong>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neuefische/ds-meetups/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>